code/obj/decal/cleanable.dm

```diff
@@ -1163,6 +1163,13 @@ var/list/blood_decal_violent_icon_states = list("floor1", "floor2", "floor3", "f
 	icon = 'icons/obj/objects.dmi'
 	icon_state = "shards"
 
+/obj/decal/cleanable/flare_off
+	name = "empty flare"
+	desc = "Used up military flare. Still not safe for human consumption."
```

Comment:
```suggestion
	desc = "A used up emergency flare. It still isn't safe for human consumption, doofus."
```
"military" flare seems a little weird to say, especially considering ss13 is a civillian outpost. Also, flares see a lot more use in emergencies than in military applications, especially the handheld kind seen here. The other rewording is up to you.

Reply:
I was trying to find a word for it to separate it from the flare gun and emergency was evading me, so I landed on "military". Thanks!